llvm/test/tools/UpdateTestChecks/update_test_checks/Inputs/difile_absolute_filenames.ll

```diff
@@ -0,0 +1,34 @@
+; RUN: opt < %s -S | FileCheck %s
+
+; ModuleID = '/home/slinder1/llvm-project/foo.c'
+source_filename = "/home/slinder1/llvm-project/foo.c"
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
I don't think we need the datalayout or triple, or function attributes?

Reply:
Right, they aren't required by any stretch, but I wanted to commit the unadulterated output of cc1, just to avoid biasing the test away from the ultimate goal for the change, which is to enable tests for CGDebugInfo related changes using update_cc_test_checks.py

I don't actually have an upstream test which will immediately benefit, and I wasn't sure how best to add a test which relies on clang but ultimately is testing UpdateTestChecks itself. If there is a better approach I'm happy to switch to it, and I am also fine stripping obviously unrelated bits from this test if I'm still not being convincing!

Reply:
I see, thanks for the background. I don't feel particularly strongly in any direction so I'm happy for you to go ahead either way. (patch still LGTM!)

Reply:
Fair enough, I can trim it down to focus just on the change, and I'll try to find another place to test that update_cc_test_checks round-trips with debug info enabled

Reply:
I tried to trim down the testcase, let me know if I missed anything!